tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -147,25 +177,26 @@ class WorksheetSuite extends BaseSuite {
 
   // From 2.13 we get `name =` part
```

Comment:
From discussions it turns out it causes a bit too verbose output and since we are not using pprint here, but toString named parameters are out. I will change the comment.